administrator/components/com_content/src/Extension/ContentComponent.php

```diff
@@ -180,6 +184,24 @@ public function getContexts(): array
         return $contexts;
     }
 
+    /**
+     * Returns valid contexts for schemaorg
+     *
+     * @return  array
+     *
+     * @since   4.0.0
```

Comment:
```suggestion
     * @since  __DEPLOY_VERSION__
```

This way it will automatically get the correct version when it is accepted, merged and released. Sorry your mentors didnt explain this